terraform-staging/govgraphsearch.tf

```diff
@@ -148,7 +148,7 @@ resource "google_cloud_run_service" "govgraphsearch" {
         }
         env {
           name  = "DISABLE_AUTH"
```

Comment:
Optional suggestion. I find positive booleans to be much clearer in the code than negative ones. Especially for feature flags.
Could be for another PR

e.g

```javascript
if (process.env.ENABLE_AUTH) {
    // do
    // auth
    // logic
}```

vs

```javascript
if (!process.env.DISABLE_AUTH) {
    // do
    // auth
    // logic
}```

Reply:
Agreed. It would avoid the double negative here. However things will get a little bit more complicated in the authentication code in the `govuk-knowledge-graph-search` repo. Worth giving it a go though.